frontend/app/src/views/settings/GeneralSettings.vue

```diff
@@ -650,40 +604,6 @@ export default class General extends Settings {
     );
   }
 
-  async onHistoricDataStartChange(date: string) {
-    const previousValue = this.parseDate(
-      this.generalSettings.historicDataStart
-    );
-
-    if (!this.notTheSame(date, previousValue)) {
-      return;
-    }
-
-    const params = {
-      date
-    };
-    const message: BaseMessage = {
-      success: `${this.$t(
-        'general_settings.validation.historic_data.success',
```

Comment:
This key also need to be removed from en.json

Reply:
done